packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -191,15 +191,22 @@ const pincodeSetScreenOptions = ({
 }: {
   route: RouteProp<StackParamList, Screens.PincodeSet>
 }) => {
-  const isVerifying = route.params?.isVerifying
-  const title = isVerifying
-    ? i18n.t('onboarding:pincodeSet.verify')
+  const changePin = route.params?.changePin
+  const title = changePin
+    ? i18n.t('onboarding:pincodeSet.changePIN')
     : i18n.t('onboarding:pincodeSet.create')
+
   return {
     ...nuxNavigationOptions,
-    headerTitle: () => (
-      <HeaderTitleWithSubtitle title={title} subTitle={i18n.t('onboarding:step', { step: '2' })} />
-    ),
+    headerTitle: () =>
```

Comment:
This is doing duplicate checks and looks a bit weird :) . Can we just leave the second option and put a space in the conditional? Something like:
```
<HeaderTitleWithSubtitle
          title={title}
          subTitle={changePin ? ' ' : i18n.t('onboarding:step', { step: '2' })}
        />
```

Reply:
agreed, changed